discretize/TensorMesh.py

```diff
@@ -160,6 +160,24 @@ def gridN(self):
         return self._getTensorGrid('N')
 
     @property
+    def gridH(self):
+        """Cell widths in grid."""
+        H = self.h
+        if self.dim == 1:
+            return H
+        if self.dim == 2:
+            H = np.meshgrid(H[0],H[1])
+            hx = np.reshape(H[0],self.nC)
+            hy = np.reshape(H[1],self.nC)
+            return np.c_[hx,hy]
+        elif self.dim == 3:
+            H = np.meshgrid(H[0],H[1],H[2])
```

Comment:
I also think that there may be some problems with the ordering on this `meshgrid`. We have `utils.ndgrid` that does things how we expect them in the vectorized form (`'F'` order). A test of uneven grids would be helpful (plotting even to make sure...).

https://github.com/simpeg/discretize/blob/dev/discretize/utils/matutils.py#L101